noxfile.py

```diff
@@ -20,22 +20,24 @@
 import glob
 import os
 import platform
+import re
 import shutil
 import shlex
+import subprocess
 import sys
 import nox
 
+
 GCC_VERSIONS = [
-    "gcc-5",
-    "gcc-6",
-    "gcc-8",
-    "gcc-9",
-    "gcc-10",
     "gcc-11",
-    "clang-10",
+    "gcc-10",
+    "gcc-9",
+    "gcc-8",
+    "gcc-6",
+    "gcc-5",
```

Comment:
No more numerical order?

Reply:
Changed from Ascending to descending to get the latest supported gcc by accessing item 0.